docs/DeveloperGuide.md

```diff
@@ -173,6 +173,32 @@ The rake symbol (in the Check validity of command action above) is used to show
 
 ### Implementation of Main Mode Features
 
+List of decks and list of cards are stored separately in `UniqueDeckList` and `UniqueCardList`. Although a card belongs to a deck, it is not stored within a deck. Each card indicates a deck that it belongs to itself.
+A deck stores its name and a Boolean value of whether it is selected or not.
+
+#### addDeck Feature
+
+`MasterDeck` extends `ReadOnlyMasterDeck`, stored internally as a `UniqueDeckList`. It implements following operation:
```

Comment:
Im not sure if its stored internally as a `UniqueDeckList`, but more like it stores a reference to UniqueDeckList?

Reply:
Yeah I think it's more correct to say so. Thx!!